### DISCUSSION TITLE
ERROR

### DISCUSSION THREAD

**Discussion by mikekap36** [community/community#36392] [Other Feature Feedback, Questions, & Ideas] [answered]
Hell there im trying to git clone a url but every time on my cmd appears the following message :
Support for password authentication was removed on... 
how can i fix that?

**Comment by airtower-luna** [comment, marked as answer]
As the message says, you cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Comment by ossanna16** [comment]
Hey there @mikekap36 and thank you for asking a good question! I'm glad you found our community 🙂
